docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,358 @@
+
+To do this, call the `WithLifetime` method and pass in `ContainerLifetime.Persistent`:
+
+```C#
+var builder = DistributedApplication.CreateBuilder(args);
+
+var queue = builder.AddRabbitMQ("rabbit")
+                   .WithLifetime(ContainerLifetime.Persistent);
+
+builder.AddProject<Projects.WebApplication1>("api")
+    .WithReference(queue).WaitFor(queue);
+
+builder.Build().Run();
+```
+
+The dashboard will show persistent containers with a pin icon:
+
+![Persistent containers](images/persistent-container.png)
+
+After the app host has stopped, the container will continue to run:
+
+![Docker desktop showing rabbit mq](images/persisent-container-dockerdesktop.png)
+
+### Resource commands
+
+The app host supports adding custom commands to resources. This is useful when you want to run custom commands on a resource. This can be done by calling the `WithCommand` method and passing in the command to run:
+
+```C#
+var builder = DistributedApplication.CreateBuilder(args);
+
+var cache = builder.AddRedis("cache")
+                   .WithClearCommand();
+
+builder.AddProject<Projects.WebApplication1>("api")
+    .WithReference(cache)
+    .WaitFor(cache);
+
+builder.Build().Run();
+```
+
+The command implementation:
+
+```C#
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using StackExchange.Redis;
+
+public static class RedisCommandExtensions
+{
+    public static IResourceBuilder<RedisResource> WithClearCommand(this IResourceBuilder<RedisResource> builder)
+    {
+        builder.WithCommand("clear-cache", "Clear Cache",
+            async context =>
+            {
+                var redisConnectionString = await builder.Resource.GetConnectionStringAsync() 
+                    ?? throw new InvalidOperationException("Unable to get the Redis connection string.");
+                
+                using var connection = ConnectionMultiplexer.Connect(redisConnectionString);
+
+                await connection.GetDatabase().ExecuteAsync("FLUSHALL");
+
+                return CommandResults.Success();
+            },
+            context =>
+            {
+                if (context.ResourceSnapshot.HealthStatus == HealthStatus.Healthy)
+                {
+                    return ResourceCommandState.Enabled;
+                }
+                return ResourceCommandState.Disabled;
+            });
+        return builder;
+    }
+}
+```
+
+These commands can be run from the dashboard:
+
+![Clear cache command on dashboard](images/clear-cache-command.png)
+
+<video controls src="videos/custom-command.mp4" title="Title"></video>
+
+### Container Networking
+
+The app host now adds all containers to a common network (`default-aspire-network`). This is useful when you want to communicate between containers without going through the host network. 
+
+This also makes it easier to migrate from docker compose to the app host, as containers can communicate with each other using the container name.
+
+### Eventing model
```

Comment:
Eventing model arrived in 8.2. The thing we did add for 9.0 is the ability to fire off async.

Reply:
It was experimental so it didn’t matter then 😁